src/os/bluestore/BlueFS.h

```diff
@@ -312,6 +389,7 @@ class BlueFS {
       const unsigned length,
       const bluefs_super_t& super);
     ceph::buffer::list::page_aligned_appender buffer_appender;  //< for const char* only
+    bufferlist::contiguous_filler* wal_header_filler; // To encode bluefs_wal_header_t we need to save the location of the header we want to fill
```

Comment:
Was it a problem to make wal_header_filler a field, not a pointer?

Reply:
Not really, I felt it was nicer than using a `contigous_filler(nullptr)`. `std::optional` might be used here too.